src/ConnectedMode/Binding/UnintrusiveBindingPathProvider.cs

```diff
@@ -30,6 +32,8 @@ namespace SonarLint.VisualStudio.ConnectedMode.Binding
     internal interface IUnintrusiveBindingPathProvider
     {
         string Get();
```

Comment:
rename to GetCurrentBindingPath or somthing like that (indicate it's the current solution somehow)